js/apps/system/_admin/aardvark/APP/react/build/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
Please adjust to ArangoDB

Reply:
Done. Keep in mind that this is public/manifest, not build/manifest